<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="lblNum1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Num1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAdd">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbClear">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNum1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNum2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblNum2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Num2</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNum3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Sum</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNum3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbExit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pbExit</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
